web/packages/teleport/src/Login/Login.tsx

```diff
@@ -78,3 +90,114 @@ export function LoginComponent({
     </>
   );
 }
+
+const LicenseBox = styled(Box)`
+  width: 100%;
+  max-width: 550px;
+  padding: ${props => props.theme.space[4]}px;
+  background-color: ${props => props.theme.colors.levels.surface};
+  border-radius: ${props => props.theme.radii[3]}px;
+  margin: auto;
+`;
+
+function LicenseAcknowledgement({
+  setLicenseAcknowledged,
+}: {
+  setLicenseAcknowledged(): void;
+}) {
+  const [checked, setChecked] = useState(false);
+
+  return (
+    <>
+      <LogoHero />
+      <LicenseBox>
+        <Text
+          css={`
+            line-height: 32px;
+            font-weight: 500;
+            font-size: ${props => props.theme.fontSizes[7]}px;
+          `}
+        >
+          Welcome to Teleport
```

Comment:
Looks like there should be a bit more padding around this title based on the design.